app/src/main/java/com/account/model/AccountDTO.java

```diff
@@ -0,0 +1,16 @@
+package com.account.model;
+
+import java.util.UUID;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
+@AllArgsConstructor
+public class AccountDTO {
```

Comment:
This doesn't reflect the expected structure. 

Body:
- `ok` - true if success, false in case of any error
- `result`
	- `id`
	- `email`
	- `username`
	- `status`
	- `createdAt`, epoch millis
- `errorMessage` - short error description